image-create-custom

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+parse_args() {
+    for arg in $@; do
+            if [[ "$arg" =~ ^rhel(-[[:digit:]]){2}(-([[:digit:]]{8}-[[:digit:]]|latest))$ ]]; then
+                image="$arg"
+                break
+            fi
+    done
+
+    if [ ! "$image" ]; then
+        echo "Missing or unsupported image (rhel-x-y-YYYYMMDD-B or rhel-x-y-latest)."
```

Comment:
Is it really wise to encode the assumptions about the name format here? This is quite fungible, it seems easier to just accept whatever the user specifies and then fail later on with a 404 not found.

Reply:
I think the format is pretty much stable, however I simplified the whole process, including removal of the check.